Makefile

```diff
@@ -15,11 +15,24 @@ check:
 check-dummy:
 	BUILD_DUMMY_WASM_BINARY= cargo check
 
+# Pseudo private target is invoked by public targets for different chains
+--try-runtime:
+	RUST_LOG=runtime=trace,try-runtime::cli=trace,executor=trace \
+		cargo run \
```

Comment:
Why do you use the `dev` profile in this case? (no --release) Is it, because you would like to allow debug_assertions? I could imagine, that the runtime speed might be too slow for complex storage migrations?